src/main/java/com/googleinterns/zoomtube/data/Line.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.googleinterns.zoomtube.data;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.repackaged.com.google.common.base.Optional;
+import com.google.auto.value.AutoValue;
+import java.util.Date;
+import javax.annotation.Nullable;
+
+/** Contains data pertaining a single line of transcript. */
+@AutoValue
+public abstract class Line {
+  public static final String ENTITY_KIND = "Line";
+  public static final String PROP_LECTURE = "lecture";
+  public static final String PROP_START = "start";
+  public static final String PROP_DURATION = "duration";
+  public static final String PROP_CONTENT = "content";
+
+  /**
+   * Creates a {@code Line} object.
+   * @param key The key for the transcript.
+   * @param lecture The key for the lecture.
+   * @param start The starting timestamp for the lecture line in seconds.
+   * @param duration The number of seconds that the timestamp lasts for.
```

Comment:
Should these be Dates or a long for number of millis?

Reply:
I'm planning on updating start to be a Date object and duration to be a long in a later pull request. Thanks for reminding me to put in a TODO.